testing/src/test/java/org/oppia/android/testing/profile/ProfileTestHelperTest.kt

```diff
@@ -102,6 +102,17 @@ class ProfileTestHelperTest {
     assertThat(profiles).hasSize(10)
   }
 
+  @Test
+  fun testAddDefaultProfile_createDefaultProfile_checkProfileIsAdded() {
+    profileTestHelper.createDefaultProfile()
+    testCoroutineDispatchers.runCurrent()
+    val profilesProvider = profileManagementController.getProfiles()
+    testCoroutineDispatchers.runCurrent()
+
+    val profiles = monitorFactory.waitForNextSuccessfulResult(profilesProvider)
+    assertThat(profiles).hasSize(1)
```

Comment:
Probably should check that it's an admin, as well.

Reply:
Done